src/domain/file/command.ts

```diff
@@ -15,9 +16,10 @@ export type StoreFileInput = {
 
 export type StoreFilesInput = StoreFileInput[]
 
-export const ResourceConflictError = () =>
+export const ResourceConflictError = (fileIds: FileId[]) =>
```

Comment:
If the error is called `ResourceConflictError`, then the argument and properties should use the naming `resource` and not `file`.